cinema/management/commands/wait_for_db.py

```diff
@@ -0,0 +1,33 @@
+import time
+
+from django.core.management.base import BaseCommand
+from django.db import connections
+from django.db.utils import OperationalError
+
+
+class Command(BaseCommand):
+    def handle(self, *args, **kwargs):
+        db_conn = None
+        max_attempts = 5
+        attempts = 0
+
+        while not db_conn and attempts < max_attempts:
+            try:
+                connections["default"].ensure_connection()
+                db_conn = connections["default"]
+            except OperationalError:
+                attempts += 1
+                self.stdout.write(
+                    f"Database unavailable, waiting 1 second... "
+                    f"(Attempt {attempts}/{max_attempts})"
+                )
+                time.sleep(1)
```

Comment:
From a code maintainability perspective, it would be better to separate the logic of establishing a database connection and the logic of handling the connection errors into different methods. This would make the `handle` method more readable and easier to maintain.